packages/shader-lab/src/codeGen/GLESVisitor.ts

```
@@ -193,9 +292,57 @@ export abstract class GLESVisitor extends CodeGenVisitor {
     return globalCode;
   }
 
-  private _getGlobalSymbol(out: ICodeSegment[]): void {
-    const { _referencedGlobals } = VisitorContext.context;
+  private _extractLocalVarNames(
+    node: ASTNode.InitDeclaratorList,
+    context: VisitorContext,
+    role: StructRole
+  ): void {
```

ESLint

[error] Replace `⏎····node:·ASTNode.InitDeclaratorList,⏎····context:·VisitorContext,⏎····role:·StructRole⏎··` with `node:·ASTNode.InitDeclaratorList,·context:·VisitorContext,·role:·StructRole`

(prettier/prettier)